R/uploadPackage.R

```diff
@@ -28,7 +28,7 @@ uploadPackageUI <- function(id) {
           tags$head(tags$script(I(paste0('$(window).on("load resize", function() {$("#', NS(id, "add_pkgs"), '").css("margin-top", $("#', NS(id, "pkg_lst"), '-label")[0].scrollHeight + .5*parseFloat(getComputedStyle(document.documentElement).fontSize));});'))))
         )
       ),
-      column(width = 1),
+      
```

Comment:
Is there a reason you removed this column? I liked the previous spacing better.

Reply:
OK I'll put it back in